### PR TITLE
enhance(ai-help): record embedding duration and model separately

### PR DESCRIPTION
**Pull request by caugner**
(MP-993)

### CODE REVIEW

File: src/ai/embeddings.rs [hunk 1]
```diff
@@ -108,44 +117,60 @@ pub async fn get_related_full_docs(
     client: &Client<OpenAIConfig>,
```
Comment (argl) [inline src/ai/embeddings.rs:117]: Not related to this specific story, but is `get_related_full_docs` even used anywhere? I could not find any reference to it.

File: src/api/ai_help.rs [hunk 2]
```diff
@@ -503,15 +506,13 @@ pub async fn ai_help(
...
-                                    response_duration: default_meta_big_int(Some(
-                                        response_duration.as_millis(),
-                                    )),
+                                    embedding_duration: default_meta_duration(ai_help_req_meta.embedding_duration),
+                                    search_duration: default_meta_duration(ai_help_req_meta.search_duration),
+                                    response_duration: default_meta_duration(Some(response_duration)),
                                     query_len: default_meta_big_int(ai_help_req_meta.query_len),
                                     context_len: default_meta_big_int(ai_help_req_meta.context_len),
                                     response_len: default_meta_big_int(Some(context.len)),
+                                    embedding_model: ai_help_req_meta.embedding_model.unwrap_or(""),
```
Comment (argl) [inline src/api/ai_help.rs:515]: nit: `unwrap_or_default()` looks a bit more idiomatic
```suggestion
                                    embedding_model: ai_help_req_meta.embedding_model.unwrap_or_default(),
```

File: src/api/ai_help.rs [hunk 3]
```diff
@@ -537,11 +538,11 @@ pub async fn ai_help(
...
                     message_id,
                     parent_id,
-                    search_duration: default_meta_big_int(
-                        ai_help_req_meta.search_duration.map(|d| d.as_millis()),
-                    ),
+                    embedding_duration: default_meta_duration(ai_help_req_meta.embedding_duration),
+                    search_duration: default_meta_duration(ai_help_req_meta.search_duration),
                     query_len: default_meta_big_int(ai_help_req_meta.query_len),
                     context_len: default_meta_big_int(ai_help_req_meta.context_len),
+                    embedding_model: ai_help_req_meta.embedding_model.unwrap_or(""),
```
Comment (argl) [inline src/api/ai_help.rs:545]: idiomatic nit:
```suggestion
embedding_model: ai_help_req_meta.embedding_model.unwrap_or_default(),
```

**Review by argl — APPROVED** [review]
Just a few cosmetics, looking good otherwise.

File: src/ai/embeddings.rs (hunk 1 above)
Comment (caugner) [reply src/ai/embeddings.rs:117]: Yes, indeed it's currently no longer used. I'll open a separate PR to remove it.

**Review by argl — APPROVED** [review]

**Pull request merged**